bundle/src/main/java/com/adobe/acs/commons/workflow/synthetic/impl/granite/SyntheticWorkflowSession.java

```diff
@@ -74,86 +65,81 @@ public SyntheticWorkflowSession(SyntheticWorkflowRunnerImpl workflowService, Ses
         if (workItem instanceof SyntheticWorkItem) {
             throw new SyntheticCompleteWorkflowException("Synthetic workflow [ "
                     + workItem.getWorkflow().getId() + " : " + workItem.getId() + " ] completed");
         } else {
-            throw new UnsupportedOperationException(UNSUPPORTED_OPERATION_MESSAGE);
+            throw new UnsupportedOperationException(UNSUPPORTED_OPERATION_MESSAGE + "complete that does not contain a SyntheticWorkItem");
         }
     }
 
-    @Override
     @SuppressWarnings("squid:S1192")
     public final void restartWorkflow(final Workflow workflow) throws WorkflowException {
         if (workflow instanceof SyntheticWorkflow) {
             throw new SyntheticRestartWorkflowException("Synthetic workflow [ " + workflow.getId() + " ] restarted");
        } else {
-            throw new UnsupportedOperationException(UNSUPPORTED_OPERATION_MESSAGE);
+            throw new UnsupportedOperationException(UNSUPPORTED_OPERATION_MESSAGE + "restartWorkflow that does not contain a SyntheticWorkflow");
         }
     }
 
-    @Override
     public final boolean isSuperuser() {
         return true;
     }
 
-    @Override
-    public final void deployModel(final WorkflowModel workflowModel) throws WorkflowException {
-        throw new UnsupportedOperationException(UNSUPPORTED_OPERATION_MESSAGE);
-    }
-
-    @Override
-    public final WorkflowModel createNewModel(final String s) throws WorkflowException {
-        throw new UnsupportedOperationException(UNSUPPORTED_OPERATION_MESSAGE);
-    }
-
-    @Override
-    public final WorkflowModel createNewModel(final String s, final String s2) throws WorkflowException {
-        throw new UnsupportedOperationException(UNSUPPORTED_OPERATION_MESSAGE);
-    }
-
-    @Override
-    public final void deleteModel(final String s) throws WorkflowException {
-        throw new UnsupportedOperationException(UNSUPPORTED_OPERATION_MESSAGE);
-    }
-
-    @Override
-    public final WorkflowModel[] getModels() throws WorkflowException {
-        throw new UnsupportedOperationException(UNSUPPORTED_OPERATION_MESSAGE);
-    }
-
-    @Override
-    public final WorkflowModel[] getModels(final WorkflowModelFilter workflowModelFilter) throws WorkflowException {
-        throw new UnsupportedOperationException(UNSUPPORTED_OPERATION_MESSAGE);
-    }
-
-    @Override
-    public final ResultSet<WorkflowModel> getModels(final long l, final long l2) throws WorkflowException {
-        throw new UnsupportedOperationException(UNSUPPORTED_OPERATION_MESSAGE);
-    }
-
-    @Override
-    public final ResultSet<WorkflowModel> getModels(final long l, final long l2,
-                                                    final WorkflowModelFilter workflowModelFilter)
-            throws WorkflowException {
-        throw new UnsupportedOperationException(UNSUPPORTED_OPERATION_MESSAGE);
+    public final Object getModel(Object[] args) throws WorkflowException {
```

Comment:
Just curious - this looks like a net-new method signature for getModel(..) was this somehow missed before..?
I dont see it in the original impl, but maybe im missing it in the diff somewhere :)

Reply:
`com.adobe.granite.workflow.WorkflowSession` has that in there, so I figured it was probably something I should cover. If we don't need it we can rip that part out.

Reply:
no worries - if it's in there, sall good. was just surprised the prior impl approach wouldve let that slip through!